app/code/Magento/Downloadable/Model/Link.php

```diff
@@ -28,6 +28,8 @@ class Link extends \Magento\Framework\Model\AbstractExtensibleModel implements C
 
     const XML_PATH_CONFIG_IS_SHAREABLE = 'catalog/downloadable/shareable';
 
+    const XML_PATH_CONFIG_STATUS_TO_ENABLE_DOWNLOAD = 'catalog/downloadable/order_item_status';
```

Comment:
The constant has been declared already:
app/code/Magento/Downloadable/Model/Link/Purchased/Item.php:49